src/stories/Library/Modals/modal-reservation/reservation.stories.tsx

```diff
@@ -37,6 +37,10 @@ export default {
       name: "State for reservation",
       defaultValue: "initial",
     },
+    showPromoBar: {
+      name: "Show PromoBar",
+      defaultValue: false,
```

Comment:
Please add a type to this control - right now it's just a JSON string. It would be nice if it was an option between yes and no/true and false
![image](https://user-images.githubusercontent.com/28546954/204747591-8a706c35-ea6b-40c6-88b8-28ad216ee6b2.png)
..like state of the reservation is -> 
![image](https://user-images.githubusercontent.com/28546954/204747728-2a3a2c97-889b-4d91-8206-e95fd5ec0252.png)